pkg/controller/plan/migration.go

```diff
@@ -860,6 +864,25 @@ func (r *Migration) execute(vm *plan.VMStatus) (err error) {
 			vm.AddError(fmt.Sprintf("Step '%s' not found", r.step(vm)))
 			break
 		}
+
+		if r.converter == nil {
+			labels := map[string]string{
```

Comment:
these two are somewhat questionable, it'd be better if they are constructed outside per provider

Reply:
+1
there is some similarity between this conversion and the guest conversion in case of warm migration from vsphere (in both cases we transfer the image(s) and then initiate another pod to convert the data we got. but they are also different in the sense that here it's per-PVC while the pod in case of warm migration should get all the volumes..
but it would be nice if we generalize this somehow and extract the warm migration to be specific to the vsphere provider, and this part specific to the OpenStack provider under a 'conversion' phase.